ldcbench.http-node/src/main/java/org/dice_research/ldcbench/nodes/http/simple/dump/comp/Archiver.java

```diff
@@ -0,0 +1,91 @@
+package org.dice_research.ldcbench.nodes.http.simple.dump.comp;
+
+import java.io.BufferedInputStream;
+import java.io.File;
+import java.io.FileInputStream;
+import java.io.IOException;
+import org.apache.commons.compress.archivers.ArchiveOutputStream;
+import org.apache.commons.compress.utils.IOUtils;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/**
+ * The abstract Archiver which defines the attributes each archiver must have
+ * and implement the logic used by all archivers to create Archives
+ *
+ */
+public abstract class Archiver {
```

Comment:
At the moment, this class fulfills two tasks.
1. It defines the methods an archiver should have.
2. It offers base implementations for these methods.

It would be better to split these two tasks. `Archiver` should be an interface that defines the methods that are necessary to be implemented by an archiver. `AbstractArchiver` should be an abstract class that offers basic implementations.